resource.go

```diff
@@ -38,6 +39,22 @@ const (
 	KindValidatingWebhookConfiguration = "ValidatingWebhookConfiguration"
 )
 
+// rank
+const (
+	RankNamespace                      = 10
+	RankServiceAccount                 = 20
```

Comment:
ServiceAccount is a namespaced resource.
Is this correct?

Reply:
Yes, I understand this.
I don't know why the rank of ServiceAccount is set to this value.
But, I keep the existing implementation.

Reply:
@terassyi Looks good if there were a comment?

Reply:
I added a comment.